MEMBER_TRAVEL_FUND.md

```diff
@@ -110,74 +110,52 @@ Once the request has been approved, you must:
-Benjamin Zaslavsky | Collab Summit Spring 2018 | Attendance - Inclusivity Initiative | Berlin, Germany | May 31 - Jun 1 2018 | $385.59
-Ben Michel | Node.js at MSBuild 2018 | Node.js Collaboration & CommComm Representation | Seattle Washington, USA | May 6 - 9 | $402
-Ujjwal Sharma | Collab Summit Spring 2018 | Attendance & Collaborate | Berlin, Germany | May 29-June 3 2018 | $1384
-Nicola Del Gobbo | Collab Summit Spring 2018 | Attendance | Berlin, DE | May 30 - June 2 2018 | €400
-Manil Chowdhury | MSBuild 2018 | Microsoft <> Node.js Collaboration, CommComm Representation | Seattle, WA, USA | May 6 - May 10 2018 | USD 209.40
-Gus Caplan | TC39 Meeting | Attendance | Redmond, WA, US | 23 July – 27 July 2018 | US $1,112.28
-Rich Trott | Collab Summit & JS Interactive 2018 | Facilitate Code & Learn | Vancouver BC CAN | Oct 10 -13 2018 | US$1270.76
-Guy Bedford | Collab Summit 2018 | Attendance & Collaboration | Vancouver, BC, CA | 12 - 13 Oct 2018 | US$2040
-Agiri Abraham Jnr | JS Interactive/Collab Summit | Attendance & Collaborate | Vancouver, BC, Cannada | Oct 10 - Oct 14 | USD 2995.00
-Dhruv Jain | Node Collab Summit & JS Interactive 2018 | Attendance, Collaborate and Code & Learn | Vancouver BC, CAN | Oct 10 - 13 2018 | $2100
-Никита Сковорода | Collab Summit 2018 | Attendance & Collaboration | Vancouver, BC, CA | 12 - 13 Oct 2018 | US$1500
-Hassan Yabagi Sani | Collab Summit & JS Interactive 2018 | Code and Learn | Vancouver BC CAN |  Oct 10 - 13 2018 | US$2741
-Ahmad Bamieh | Collab Summit & JS Interactive 2018 | Attendance, Collaborate and Code & Learn | Vancouver BC CAN |  Oct 10 - 13 2018 | US$2250
-Patricia Realini | Node Summit | Panel Speaker | San Francisco, CA, USA | July 23-25 2018 | USD$1150
-Alejandro Oviedo | Collab Summit & JS Interactive 2018 | Attendance & Collaborate | Vancouver, BC, Canada | Oct 10 - Oct 14 | USD 2200
-Anna Henningsen | Collab Summit & JS Interactive 2018 | Attendance, Collaborate and Code & Learn | Vancouver BC CAN |  Oct 10 - 13 2018 | US$1360
-Ujjwal Sharma | Collab Summit & JS Interactive 2018 | Attendance & Collaborate | Vacouver BC, CAN | Oct 9 - Oct 14 | $2500
-Agiri Abraham Jnr | ConcatenateConf | Attendance & Manning Booth | Lagos, Nigiera | AUG 10 - AUG 11 | USD 393
-Jamie Davis | Collab Summit & JS Interactive 2018 | Attendance & Collaborate | Vancouver BC, CAN | Oct 9 - Oct 14 | US$1175
-Rachel White | Collab Summit & JS Interactive 2018 | Attendance & Collaborate | Vancouver BC, CAN | Oct 9 - Oct 14 | US $2439.86
-Adam Miller | Collab Summit & JS Interactive 2018 | Attendance & Collaborate | Vancouver BC, CAN | Oct 9 - Oct 14 | US$2115
-Trivikram Kamat | Collab Summit & JS Interactive 2018 | Attendance, Collaborate and Code & Learn | Vancouver BC CAN |  Oct 10 - 13 2018 | US$1000
-Tobias Nießen | Collab Summit & JS Interactive 2018 | Attendance, Collaborate and Code & Learn | Vancouver, BC, CAN | Oct 9 - Oct 14 2018 | US$2100
```

Comment:
@ladyleet this new list doesn't include all previous entries.
Also, in new data, I see many entries have blank fields for
"Date Expense report sent", 
"Amount of Expense Report",
"Bill.com Amount approved for reimbursement". etc

Reply:
Thank you so much for doing this! My eyes bled out a little trying to reconcile everything. I'll include these changes as well as wait for others' feedback before doing so.

Yes - the blank fields are because those are updated as expense reports get sent. :)